build.sbt

```diff
@@ -206,110 +204,51 @@ lazy val server: Project = (project in file("server"))
   ))
   .dependsOn(core, restSqs, commonTest % "test")
 
-val dockerGraalvmNative = taskKey[Unit]("Create a docker image containing a binary build with GraalVM's native-image.")
-val dockerGraalvmNativeImageName =
-  settingKey[String]("Name of the generated docker image, containing the native binary.")
-
 lazy val nativeServer: Project = (project in file("native-server"))
-  .enablePlugins(JavaServerAppPackaging, DockerPlugin)
+  .enablePlugins(GraalVMNativeImagePlugin, DockerPlugin)
   .settings(buildSettings)
   .settings(Seq(
     name := "elasticmq-native-server",
     libraryDependencies += "com.oracle.substratevm" % "svm" % "19.2.1" % Provided,
```

Comment:
shouldn't this version match the `graalVMNativeImageGraalVersion`?

Reply:
It's interesting, because apparently there is a vm-20.0.0 tag on github, but the latest artifact in Maven Central is 19.2.1